src/DIRAC/Core/Base/AgentModule.py

```diff
@@ -329,6 +329,7 @@ def _setShifterProxy(self):
 
     def am_go(self):
         # Set the shifter proxy if required
+        initialWallTime = initialCPUTime = cpuPercentage = -1
```

Comment:
I'm not sure I like adding this for the sake of a linter...

I think I'd prefer refactoring it like this:

```diff
         if self.activityMonitoring:
             initialWallTime, initialCPUTime, mem = self._startReportToMonitoring()
-        cycleResult = self.__executeModuleCycle()
-        if self.activityMonitoring and initialWallTime and initialCPUTime:
+            cycleResult = self.__executeModuleCycle()
             cpuPercentage = self._endReportToMonitoring(initialWallTime, initialCPUTime)
+.       else:
+            cycleResult = self.__executeModuleCycle()
```

Reply:
Works with me. I am surprise that pylint does not have a false positive on that one though :-)